src/shared/utils/utils.ts

```diff
@@ -77,11 +78,11 @@ export const isWrappedV3Document = (
 ): document is WrappedDocumentV3<OpenAttestationDocumentV3> => {
   return document && document.version === SchemaId.v3;
 };
-export const isWrappedV2Document = (
-  document: WrappedDocument<OpenAttestationDocument>
-): document is WrappedDocumentV2<OpenAttestationDocumentV2> => {
-  return !isWrappedV3Document(document);
+
+export const isWrappedV2Document = (document: any): document is WrappedDocumentV2<OpenAttestationDocumentV2> => {
+  return !!(document && validate(document, getSchema(SchemaId.v2)).length === 0 && Signature.guard(document.signature));
```

Comment:
possible to give a readable name for this boolean?
`validate(document, getSchema(SchemaId.v2)).length === 0`

Reply:
dont understand ?

Reply:
its fine for me, but it'll be easier to parse for future people imo.

its a nitpick, I'm fine with this

Reply:
y'all talking past each other

zl is asking to extract the boolean conditional check

laurent is saying he doesn't understand

Reply:
done